ansible_base/lib/dynamic_config/settings_logic.py

```diff
@@ -283,6 +283,17 @@ def get_dab_settings(
 
         dab_data['ALLOW_OAUTH2_FOR_EXTERNAL_USERS'] = False
 
+        dab_data['ANSIBLE_BASE_OAUTH2_PROVIDER_PERMISSIONS_CHECK_IGNORED_VIEWS'] = [
+            'drf_spectacular.views.SpectacularAPIView',
+            'drf_spectacular.views.SpectacularRedocView',
+            'drf_spectacular.views.SpectacularSwaggerView',
+            'ansible_base.oauth2_provider.views.authorization_root.ApiOAuthAuthorizationRootView',
```

Comment:
I'd like to try an minimize the need to put views in this setting. If possible it would be nice for the check to just ignore AllowAny views rather than having to add them to this list.

Reply:
Also, I think we should change this so that there are the default DAB views that will be ignored and then a service can add additional views it wants to ignore but it wouldn't be able to delete the items from DAB itself.

Reply:
I've got it down to 0 for DAB views, split the setttings into `ANSIBLE_BASE_OAUTH2_PROVIDER_DEFAULT_IGNORED_VIEWS` (for use in dab) and `ANSIBLE_BASE_OAUTH2_PROVIDER_IGNORED_VIEWS` (for use in applications consuming dab).